packages/kbn-discover-utils/index.ts

```diff
@@ -44,3 +44,4 @@ export {
 } from './src';
 
 export * from './src/types';
+export * from './src/services/features_registry';
```

Comment:
Hi @tonyghiani,

Thanks for introducing a feature registry for Discover! It's a great way to prevent circular deps.

I would suggest to create a new package for this new service and name it the same way as the new plugin. Like `discover-shared` or `discover-registry`.

Reply:
Hi @jughosta!

I considered this option as well and I've seen this pattern applied for the unified doc viewer package & plugin.
I created this on this discover utils package to avoid too many dependencies and since it's tres-shakeable, do you see any additional benefit in moving this utility into a different package?

P.S. Regarding the unified doc viewer package, it makes to have it in its package as it also holds the UI elements to create a more complex element and is consumed by different plugins, while this registry is ideally only used by discoverShared.

Reply:
For the separation of concerns it looks cleaner to me to have a separate package.

Reply:
I've implemented a new package for the registry in [5543214](https://github.com/tonyghiani/kibana/pull/1/commits/5543214b18e367dedb5ebf06403fa98c87719845), let me know if looks better now :)